docs/install.md

```diff
@@ -43,7 +43,7 @@ CFLAGS="${CFLAGS} -mavx2" pip install --upgrade --no-cache-dir --force-reinstall
 conda install -y jpeg libtiff
 ```
 
-**Note**:  You need to run `pip uninstall mmcv` first if you have mmcv installed.
+**Note**:  You need to run `pip uninstall mmcv (mmcv-full)` first if you have mmcv (mmcv-full) installed.
```

Comment:
Does this mean that the user need to re-install mmcv-full?

Reply:
If the user intends to install mmcv, then mmcv-full should be uninstalled firstly.
Or it can be explained in more detail here

Reply:
Please explain it with more detail

Reply:
Maybe the current version is enough to understand? (I have pushed a new commit half an hour ago)

Reply:
mmcv-full is preferred for this repo, so let's ignore the replacing of `mmcv-full` with `mmcv` case.

Reply:
ok